.github/workflows/tests_linux_x86_nvidia_gpu.yml

```diff
@@ -0,0 +1,107 @@
+name: Tests::Linux::x86_64::Nvidia_GPU
+on:
+  release:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  COVERAGE_FLAGS: "--cov=pennylane_lightning_kokkos --cov-report=term-missing --cov-report=xml:./coverage.xml --no-flaky-report -p no:warnings --tb=native" 
+  GCC_VERSION: 11
+  CI_CUDA_ARCH: 86
+
+concurrency:
+  group: gpu-test-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  cpptests:
+    runs-on:
+      - self-hosted
+      - ubuntu-22.04
+      - gpu
+
+    strategy:
+      max-parallel: 1
+
+    steps:
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.8'
+
+      - name: Remove Ubuntu unattended upgrades
+        run: sudo apt-get remove -y -q unattended-upgrades
+
+      - name: Install required packages
+        run: |
+          sudo apt-get update && sudo apt-get -y -q install gcc-${{ env.GCC_VERSION }} g++-${{ env.GCC_VERSION }} gcovr lcov
+          python -m pip install ninja cmake~=3.24.3
+      - name: Checkout PennyLane-Lightning-Kokkos
+        uses: actions/checkout@v3
+
+      - name: Validate GPU version and installed compiler
```

Comment:
Should we add a stopping condition here since we want to validate the GPU version?

Reply:
Thanks @vincentmr !  The commands below are used to validate if gpu driver and compiler is available. If not, there will be errors.